.github/workflows/ark.unit.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
         with:
           version: v1.61
           working-directory: ./pkg/client-sdk
-          args: --timeout 5m
+          args: --timeout 5m --skip-files=.*_test.go
```

Comment:
we're skipping all test files this way while we want to skip only the wasm ones?

Reply:
do we want linting of test files?